packages/babel-helper-remap-async-to-generator/src/index.ts

```diff
@@ -13,26 +13,23 @@ const {
   yieldExpression,
 } = t;
 
-const awaitVisitor = traverse.visitors.merge<{ wrapAwait: t.Expression }>([
-  {
-    ArrowFunctionExpression(path) {
-      path.skip();
-    },
+const awaitVisitor = visitors.environmentVisitor<{ wrapAwait: t.Expression }>({
```

Comment:
_traverse.visitors.environmentVisitor is not a function

Reply:
Hi, Yes, this PR cause an eslint issue in our codebase repo, this is the version we are on. 


<img width="479" alt="Screenshot 2024-07-30 at 8 59 06 AM" src="https://github.com/user-attachments/assets/725c9b1f-8a22-4a32-9399-8db9ea985b57">

Is there an issue opened or plans to fix this? Thanks! 

@nicolo-ribaudo